client/src/common/utils/localStorage.js

```diff
@@ -0,0 +1,22 @@
+export const loadSettings = () => {
+  try {
+    const serializedState = localStorage.getItem('settings');
+
+    if (serializedState === null) {
+      return undefined;
+    }
+
+    return JSON.parse(serializedState);
+  } catch (err) {
```

Comment:
Consider at least loggin the error out.

Reply:
we avoid logging anything on the front, and we don't have now any functionality to save some logs for future debugging